README.md

```diff
@@ -91,21 +75,6 @@ $ sudo su
 # docker push lampepfl/dotty:$(date +%F)
 ```
 
-The new image should now appear in <https://hub.docker.com/r/lampepfl/dotty/tags/>. The next step is to open a PR againts <https://github.com/lampepfl/dotty> to change the tag of the docker image in `.drone.yml`.
-
-### Creating the docker image on macOS ###
```

Comment:
This is no longer true. I've just installed docker from https://docs.docker.com/docker-for-mac/install/ and was able to run the previous commands.